api/api/specs/routers.yaml

```diff
@@ -655,6 +666,8 @@ components:
         - port
         - env
       properties:
+        id:
+          type: "integer"
```

Comment:
Thanks for bringing that up, should a similar change be applied to the `id` (an integer) in `Event`, `Ensembler` and `Router` too?

Reply:
For consistency, yes, however it's not crucial. I think the reason why `type: "integer"` is used somewhere is because it's impossible to use `readOnly: true` together with `$ref`